qiime/metadata.py

```diff
@@ -14,12 +16,33 @@ def __init__(self, dataframe):
     def __repr__(self):
         return self._dataframe.to_json()
 
+    @staticmethod
+    def load(path):
+        try:
+            df = pd.read_csv(path, sep='\t', dtype=object).\
```

Comment:
Thanks. Just want to confirm that we are fine allowing empty columns/rows, which are a problem when working in Excel and I'm not sure if Google spreadsheets also sends these.